src/twisted/web/static.py

```diff
@@ -567,9 +570,15 @@ def _setContentHeaders(self, request, size=None):
             size = self.getFileSize()
         request.setHeader(b'content-length', intToBytes(size))
         if self.type:
-            request.setHeader(b'content-type', networkString(self.type))
+            contentType = self.type
+            if isinstance(contentType, unicode):
+                contentType = contentType.encode("utf-8")
```

Comment:
Header values should be ASCII.  Also, is the `type` instance variable supposed to allow native strings?  Is that why this can be unicode?